src/com/google/enterprise/adaptor/ValidatedUri.java

```diff
@@ -60,7 +60,10 @@ public ValidatedUri(String uriString) throws URISyntaxException {
       uri = new URI(uriString);
       url = uri.toURL();
     } catch (MalformedURLException e) {
-      throw new URISyntaxException(uriString, e.getMessage());
+      int index = e.getMessage().indexOf(": ");
```

Comment:
This feels awkward, and probably produces a nicer error message, but I do not like throwing away info when logging an error. It'd be just our luck the message is "InterestingException: null".